cmake-modules/AddGoogleTest.cmake

```diff
@@ -15,7 +15,7 @@ if(CMAKE_VERSION VERSION_LESS 3.11)
     download_project(
         PROJ                googletest
 		GIT_REPOSITORY      https://github.com/google/googletest.git
-		GIT_TAG             release-1.10.0
+		GIT_TAG             release-1.11.0
```

Comment:
Can you update the cgmanifest as well since we are updating this?

https://github.com/google/googletest/releases/tag/release-1.11.0

https://github.com/Azure/azure-sdk-for-cpp/blob/d2fd862ce1b29401ad41566fe73ae6ecb3734b72/sdk/core/azure-core/cgmanifest.json#L23-L32